<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRightWidget</class>
 <widget class="QWidget" name="QRightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="55,1">
   <item>
    <widget class="QSelfPlainTextEdit" name="plainTextEdit">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(250,250,250);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Rowlabel">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>0</y>
        <width>30</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Row：</string>
      </property>
     </widget>
     <widget class="QLabel" name="RowVlalueLabe">
      <property name="geometry">
       <rect>
        <x>663</x>
        <y>0</y>
        <width>60</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ColLable">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>0</y>
        <width>30</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Col：</string>
      </property>
     </widget>
     <widget class="QLabel" name="CloValue">
      <property name="geometry">
       <rect>
        <x>773</x>
        <y>0</y>
        <width>60</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="CodecLable">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>0</y>
        <width>35</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Codec：</string>
      </property>
     </widget>
     <widget class="QLabel" name="CodecLableValue">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>0</y>
        <width>61</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSelfPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qselfplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
